run_dir/static/js/controls.js

```diff
@@ -0,0 +1,66 @@
+/*
+File: controls.js
+URL: /static/js/controls.js
+Powers /controls/[List type] - template is run_dir/design/controls.html
+*/
+
+$( document ).ready(function() {
+  document.title = 'something else';
+  init_listjs_controls('negative_controls_table');
+});
+
+// Initialize sorting and searching javascript plugin 
+function init_listjs_controls(table_name) {
```

Comment:
Change function name to `init_datatable_controls` since we don't use listjs anymore.